src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const basicDiscount = 20;
+  const extraDiscount = 50;
+
+  if (days >= 7) {
+    return (days * dayCost) - extraDiscount;
+  }
+
+  if (days >= 3) {
+    return (days * dayCost) - basicDiscount;
+  }
+
+  return (days * dayCost);
```

Comment:
Don't repeat yourself: you've calculated `days * dayCost` 3 times, let's do it only once